js/main.js

```diff
@@ -0,0 +1,31 @@
+$(document).ready(function() {
+  $.ajax({
+      url: 'http://localhost:3000/articles',
+      type: 'GET'
+    }).done(ArticleApp.getArticles);
```

Comment:
I like that you're calling a reusable function here instead of just an anonymous function.